### ISSUE TITLE
About 'context word embedding'  in contexe2vec

### ISSUE THREAD

**Issue by daf11865** [orenmel/context2vec#3]
Hi, 
firstly sorry because I'm not a user of chainer, so want to directly ask this. 
I'm just curious about did you train 'context word embedding' with initializing a lookup table or use other pre-trained word2vec without further fine-tuning?

I think it is the former, am I right?

**Comment by orenmel** [comment]
Indeed, I used a random initialization and then trained the context word embeddings as part of the training of the entire model.

**Comment by daf11865** [comment]
thank you!
